docs/styling/THEME.md

```diff
@@ -256,7 +256,7 @@ as a parameter:
 ```scss
 // _styling.scss
 @use '@angular/material' as mat;
-@use '@o3r/styling' as o3r;
+@use '@o3r/styling/otter-theme' as otter-theme;
```

Comment:
```suggestion
@use '@o3r/styling/otter-theme';
```
We don't need the `as` if we don't rename it :)

Reply:
That's true but I keep it as good practice